### PR TITLE
Fix ConfiguredName

### PR DESCRIPTION
**Pull request by donavanbecker**
From what I can see `ConfiguredName` should be setup like this.

Also added a log to indicate if the name is empty completely.

### CODE REVIEW

**Review by bauer-andreas — DISMISSED** [review]
Great catch 🚀

File: src/lib/util/checkName.ts [hunk 1]
```diff
@@ -6,6 +6,11 @@
 
 // eslint-disable-next-line @typescript-eslint/no-explicit-any, @typescript-eslint/explicit-module-boundary-types
 export function checkName(displayName: string, name: string, value: any): void {
+  if (!value) {
```
Comment (donavanbecker) [inline src/lib/util/checkName.ts]: Is this log needed? @Supereg

**Review by hjdhjd — DISMISSED** [review]
Only suggestion would be to ensure the verbiage is consistent with other warning/error messages in format/wording (I presume it is?).

File: src/lib/util/checkName.ts (hunk 1 above)
Comment (donavanbecker) [reply src/lib/util/checkName.ts]: @hjdhjd Do you think this new log is needed?

**Comment by donavanbecker** [comment]
> Only suggestion would be to ensure the verbiage is consistent with other warning/error messages in format/wording (I presume it is?).

I have removed the log and added the validation to the check.

**Pull request merged**